docs/content/latest/architecture/docdb-sharding/sharding.md

```diff
@@ -34,13 +34,11 @@ With (consistent) hash sharding, data is evenly and randomly distributed across
 
 ![tablet_hash_1](/images/architecture/tablet_hash_1.png)
 
-The hash space for hash-sharded YugabyteDB tables is the 2-byte range from 0x0000 to 0xFFFF. Such
-a table may therefore have at most 64K tablets. We expect this to be sufficient in practice even for
-very large data sets or cluster sizes. As an example, for a table with 16 tablets the overall hash space [0x0000 to 0xFFFF) is divided into 16 sub-ranges, one for each tablet:  [0x0000, 0x1000), [0x1000, 0x2000), … , [0xF000, 0xFFFF]. Read and write operations are processed by converting the primary key into an internal key and its hash value, and determining what tablet the operation should be routed to. The figure below illustrates this.
+The hash space for hash-sharded YugabyteDB tables is the 2-byte range from 0x0000 to 0xFFFF. Such a table may therefore have at most 64K tablets. This should be sufficient in practice even for very large data sets or cluster sizes. As an example, for a table with 16 tablets the overall hash space [0x0000 to 0xFFFF) is divided into 16 sub-ranges, one for each tablet: [0x0000, 0x1000), [0x1000, 0x2000), … , [0xF000, 0xFFFF]. Read and write operations are processed by converting the primary key into an internal key and its hash value, and determining what tablet the operation should be routed to. The figure below illustrates this.
 
 ![tablet_hash](/images/architecture/tablet_hash.png)
 
-The insert/update/upsert by the end user is processed by serializing and hashing the primary key into byte-sequences and determining the tablet they belong to. Let us assume that the user is trying to insert a key k with a value v into a table T. The figure below illustrates how the tablet owning the key for the above table is determined.
+The insert/update/upsert by the end user is processed by serializing and hashing the primary key into byte-sequences and determining the tablet they belong to. Assume that a user is trying to insert a key k with a value v into a table T. The figure below illustrates how the tablet owning the key for the above table is determined.
```

Comment:
The figure below illustrates how the tablet owning the key for the above table is determined. --> The following illustration demonstrates how the tablet owning the key for the above table is determined: